### DISCUSSION TITLE
cant accessGlances Web User Interface started on http://127.0.0.1:61209/

### DISCUSSION THREAD

**Discussion by nutsofamily** [nicolargo/glances#2257] [General]
Installed Glances in Home |Assistant os 9.x addons pi4.
Started properly, states "Glances Web User Interface started on http://127.0.0.1:61209/"

That url is inaccessible. 
Substituting 127.0... with Hassio ip does not work.
From webui Glances works properly and disp;ays all information.

Therefore cannot add Glances info into any lovelace cards

**Comment by RazCrimson** [comment]
If you host glances on `127.0.0.1` then you can access the Web UI only from the Pi4. If you try to access it from some other device then it would fail as the access is limited just to the localhost IP (`127.0.0.1`).

The simplest solution would be to make glances bind to `0.0.0.0` which would accept all the connections. If you have afixed IP assigned to your home server you can use that instead. CLI options to do that would be as follows:
```
glances -w --bind 0.0.0.0
```
I am not sure how you are starting glances, but if you are using systemd, you can add the `--bind` option along with the IP to the [service file](https://github.com/nicolargo/glances/wiki/Start-Glances-through-Systemd) mentioned.

**Reply by nutsofamily** [reply]
Thanks. I’m running Hassio not Raspian on Pi4 so glances installed through addons and observed via webui

**Reply by RazCrimson** [reply]
@nutsofamily 
The Hassio page seems to ask you to manually start glances with `sudo glances -w`
Ref: [ https://www.home-assistant.io/integrations/glances/]( https://www.home-assistant.io/integrations/glances/)

If you are not starting glances as mentioned in the docs, then you need to find out how that is being done and add the above mentioned parameter to fix your issue.
